src/main/resources/defaults.yaml

```diff
@@ -171,15 +171,22 @@ all:
   - z-index: auto
 
 a:
+  - border-image-width: 1
```

Comment:
This occurs multiple times. Is there a way to define it only once? I'm not that familiar with the current `defaults.yaml`.